test/server/test_image_validation.py

```diff
@@ -25,59 +19,103 @@ def user_setup():
         return (shot_data, shot_json, user)
 
 
-def test_invalid_data_image():
-    image = image_setup()
+def test_invalid_png_data_image():
+    image = example_images['example_images'][0]['url']
     (shot_data, shot_json, user) = user_setup()
     valid_header = "data:image/png;base64,iVBORw0KGgo"
-    invalid_header = "data:image/png;base64,R0k"
-    if valid_header in image:
-        for clip_id in shot_json['clips']:
-            shot_json['clips'][clip_id]['image']['url'] = invalid_header
-            break
-
-        resp = user.session.put(
-            shot_data,
-            json=shot_json,
-        )
-        print(resp.text)
-        assert resp.status_code == 500
-
-
-def test_invalid_data_image_decoded():
-    image = image_setup()
+    invalid_header = 'data:image/png;base64,R0k'
+    assert valid_header in image
+
+    for clip_id in shot_json['clips']:
+        shot_json['clips'][clip_id]['image']['url'] = invalid_header
+        break
+
+    resp = user.session.put(shot_data, json=shot_json,)
+    print(resp.text)
+    assert resp.status_code == 500
+
+
+def test_invalid_png_data_image_decoded():
+    image = example_images['example_images'][0]['url']
```

Comment:
For this test and others I was getting a key error and needed to replace `example_images['example_images']` with `example_images` to fix it.